docs/markdown/android/switch.md

```diff
@@ -0,0 +1,129 @@
+    <Dont title="Don't" />
+    Use checkboxes or radio buttons to replace the functionality of a switch. If the functionality is a binary on or off, use Switch instead.
+  </Group>
+</TwoCol>
+
+## Accessibility
+
+Switches should have [Labels](https://gestalt.pinterest.systems/web/label) that can be read by screen readers, and that can be tapped to make it easier for users to select and deselect.
+People use Apple and Android’s accessibility features, such as VoiceOver and TalkBack to personalize how they interact with their device. Supporting these personalizations ensures that everyone has a great user experience. See Apple and Android’s accessibility documentation for more information:
+<TwoCol>
+  <Group>
+    [Accessible design on iOS](https://developer.apple.com/design/human-interface-guidelines/accessibility/overview/introduction/)
+    [Accessible development on iOS](https://developer.apple.com/accessibility/ios/)
+  </Group>
+
+  <Group>
+    [Accessible design on Android](https://material.io/design/usability/accessibility.html#understanding-accessibility)
+    [Accessible development on Android](https://developer.android.com/guide/topics/ui/accessibility)
+  </Group>
+</TwoCol>
+
+## Writing
+
+<TwoCol>
+  <Group>
+    <Do title="Do" />
+    - Be clear and brief with Switch labels so they can be easily understood.
+    - When possible, use verbs to clarify the action. Something like “set…” or “show…”.
+    - If possible, be clear whether the setting is activated or deactivated.
+    - Use sentence case for labels.
+  </Group>
+
+  <Group>
+    <Dont title="Don't" />
+    - Use vague language out of context, like “turn on” or “turn off” repeating the state of the switch is redundant and can clutter the interface.
+    - Don’t use “you,” “your,” or “my” to describe an action. Instead of “turn on your notifications,” say “turn on notifications.”
+  </Group>
+</TwoCol>
+
+## Variants
+
+### Disabled and active states
+
+1. **Default | Inactive state**
+  Switch is “off” and is not disabled
+1. **Default | Active state**
+  Switch is “on” and is not disabled
+1. **Disabled | Inactive state**
+  Switch is “off” and is disabled
+1. **Disabled | Active state**
+  Switch is “on” and is disabled
+
+<TwoCol>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/7f/6b/5a/7f6b5a6438b9331f9172d346715cdd0c.jpg" noPadding alt="Example of default inactive state of Switch" />
+    **Default | Inactive state**
+  </Group>
+
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/c0/4c/97/c04c970f0d926842cf4ef89ad42b9347.jpg" noPadding alt="Example of disabled inactive state of Switch" />
+    **Disabled | Inactive state**
+  </Group>
+
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/80/29/45/802945dd6c7a4dcf2eb43bc4d1cc9f3f.jpg" noPadding alt="Example of default active state of Switch" />
+    **Default | Active state**
+  </Group>
```

Comment:
I would reverse these two, I'd expect both active states first (also matches how it's listed above)